src/modules/events/EventsList.tsx

```diff
@@ -0,0 +1,31 @@
+import PagesWrapper from "@/components/layout/PagesWrapper";
+import ButtonLink from "@/components/ui/buttons/ButtonLink";
+import TitleHeader from "@/components/ui/TitleHeader";
+import * as texts from "@/localisation/events";
+import EventCard from "@/modules/events/EventsCard";
+import { blogPosts } from "@/data/blogsData";
+
+
+const EventsPage = () => {
+    return (
+        <div className="flex flex-col">
+            <PagesWrapper>
+                <TitleHeader
+                    title="Upcoming Events"
+                    description="Explore the upcoming events and find something that interests you. You can register for any events you'd like to attend or add new ones that you think others might enjoy. Stay involved and make the most of the opportunities available!"
+                />
+                <div className="grid grid-cols-1 gap-4 md:grid-cols-3 lg:grid-cols-4">
+                    {blogPosts.map((post, index) => (
```

Comment:
Are you using blogs as an event ? Might create a confusion.

Reply:
Oh yes! I was using dummy blog data as dummy data  until we have the actual event data to replace it.

Reply:
But you have to create your own dummy data with the `Event` type ! Blog data is of something different type and TS might be returning error. Also it could be the same build issue on pipleline as it says fail down below.